googler

```diff
@@ -334,6 +334,15 @@ except GetoptError as e:
     print("googler:", e)
     sys.exit(1)
 
+# Set up logging
```

Comment:
Frankly, I don't want to execute a single line of code if it's not necessary. The debug mode is for troubleshooting, which I hope is not the regular case with `googler`. As I mentioned earlier, the response body is quite useful but I think at this stage of the project (reasonably matured), and with the few lines of code that it has, introducing a full framework for logging is a overkill. Because it adds to maintenance too. When they changed print from 2.7 to 3.x we needed to adapt to it everywhere. Probably we should avoid that in future by adding features which aren't indispensable.